internal/cmd/clone/clone.go

```diff
@@ -19,14 +43,32 @@ func (o *options) AddFlags(cmd *cobra.Command) {
 		"",
 		"specify branch to check out",
 	)
+	cmd.Flags().VarP(
+		&o.expectedRootKeys,
+		"root-key-path",
+		"r",
```

Comment:
Let's drop `VarP` to just `Var` and remove the shorthand. This is a command that directly shadows a git command, so I don't want us to use a flag shorthand that may eventually collide with git itself.

Reply:
Fixed